src/components/ProfitAndLossSummaries/internal/profit_and_loss_summaries_summary.scss

```diff
@@ -0,0 +1,56 @@
+.Layer__ProfitAndLossSummariesSummary {
+  display: grid;
+  min-block-size: 3rem;
+  grid-template-columns: auto minmax(0, 1fr);
+  column-gap: var(--spacing-xs);
+  row-gap: var(--spacing-3xs);
+  grid-template-areas:
+    'chart heading'
+    'chart amount'
+  ;
+
+  &[data-size="lg"] {
```

Comment:
I like this pattern because I believe to support a new size, we'd just need to add it into the possible prop values and then modify _this one css file_ to support it. Is that right?

Reply:
Yes, exactly.

It's non-negotiable that we co-locate our styling. When you change the behavior of one CSS class, it should be extremely obvious that you are changing the appearance of exactly one component.